migrations.lock

```diff
@@ -606,6 +606,7 @@ geospatial
  0006_geoconfig_max_cases_per_user_and_more
  0007_geoconfig_max_case_distance_and_more
  0008_geoconfig_flag_assigned_cases
+ 0009_geoconfig_api_key_cbc_encryption
```

Comment:
This is from a merged PR?

Reply:
Yes that was a mistake, deleted here [b2dae2e](https://github.com/dimagi/commcare-hq/pull/35666/commits/b2dae2ed30fab286eecdb5a76afa3af6cd31fa0c)